code/obj/item/clothing/gloves.dm

```diff
@@ -508,15 +509,16 @@ ABSTRACT_TYPE(/obj/item/clothing/gloves)
 	name = "transparent gloves"
 	icon_state = "transparent"
 	item_state = "transparent"
-	material_prints = "transparent high-quality synthetic fibers"
+	material_prints = "insulative fibers"
+	no_prints = TRUE
```

Comment:
This is meant to be removed, no?

Reply:
as it stands what it does is make you need to remove the gloves before scanning which prints belong to it